settings.py

```diff
@@ -1612,3 +1612,7 @@
     pass
 else:
     initialize(DATADOG_API_KEY, DATADOG_APP_KEY)
+
+REST_FRAMEWORK = {
```

Comment:
right. thanks, didn't realize this affected that

On Tue, Feb 2, 2016 at 12:46 AM, Simon Kelly <notifications@github.com>
wrote:

> In settings.py
> <https://github.com/dimagi/commcare-hq/pull/10146#discussion_r51526866>:
>
> > @@ -1612,3 +1612,7 @@
> >      pass
> >  else:
> >      initialize(DATADOG_API_KEY, DATADOG_APP_KEY)
> > +
> > +REST_FRAMEWORK = {
>
> We use the Django Rest Framework for serializing the SQL forms and cases.
> This sets the default date format.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/10146/files#r51526866>.
>